### PR TITLE
Appealsv2 fillintheblanks mg

### PR DESCRIPTION
**Pull request by markgreenburg**
Replaces a bunch of placeholders with actual properties from the api response.

### CODE REVIEW

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 1]
```diff
@@ -1048,42 +1076,47 @@ export function getAlertContent(alert) {
...
+    }
+    case ALERT_TYPES.scheduledHearing: {
+      const formattedDate = moment(alert.date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
       return {
-        title: 'A hearing has been scheduled',
-        description: (
-          <p>Your [TYPE] hearing has been scheduled for [DATE] at [LOCATION].</p>
-        ),
+        title: `Your hearing is scheduled for ${formattedDate}`,
+        description: null,
```
Comment (cvalarida) [inline src/js/claims-status/utils/appeals-v2-helpers.jsx]: Where are you getting this title and description change from?

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 2]
```diff
@@ -1048,42 +1076,47 @@ export function getAlertContent(alert) {
...
         displayType: 'take_action',
         type
       };
-    case ALERT_TYPES.hearingNoShow:
+    }
+    case ALERT_TYPES.hearingNoShow: {
+      const formattedDate = moment(alert.date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
```
Comment (cvalarida) [inline src/js/claims-status/utils/appeals-v2-helpers.jsx]: Where are you getting these detail fields from to know this should be `alert.date`? Is there a list somewhere?

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 3]
```diff
@@ -1048,42 +1076,47 @@ export function getAlertContent(alert) {
...
+      // const formattedDueDate = moment(alert.date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
```
Comment (cvalarida) [inline src/js/claims-status/utils/appeals-v2-helpers.jsx]: Probably can nix this.

**Review by cvalarida — COMMENTED** [review]
Looks good. I'd just like to get confirmation on a couple things before I give this the :+1:

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 4]
```diff
@@ -466,14 +479,17 @@ export function getStatusContents(statusType, details) {
         Veteran Service Organization or representative for more information.</p>
       );
       break;
-    case STATUS_TYPES.death:
+    case STATUS_TYPES.death: {
+      const { first, middle, last } = name;
+      const nameString = `${first || ''} ${middle || ''} ${last || ''}`;
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: For consistency, mirrors the way names are displayed in the 'My Profile' section.

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 5]
```diff
@@ -466,14 +479,17 @@ export function getStatusContents(statusType, details) {
...
       contents.title = 'The appeal was closed';
       contents.description = (
-        <p>VA records indicate that [VETERAN NAME] is deceased, so this appeal has been closed. If
+        <p>VA records indicate that {_.startCase(_.toLower(nameString))} is deceased, so this appeal has been closed. If
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: Me neither :) Same as above - this is the way names are displayed on the profile page.

File: src/js/claims-status/utils/appeals-v2-helpers.jsx (hunk 1 above)
Comment (markgreenburg) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: Chris G responded to some lingering questions I had on the linked issue.
Tl;dr is that this is the current content, and it may change somewhat next week.

File: src/js/claims-status/utils/appeals-v2-helpers.jsx (hunk 2 above)
Comment (markgreenburg) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: Yep, from the caseflow api progress ticket:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7023

File: src/js/claims-status/utils/appeals-v2-helpers.jsx (hunk 3 above)
Comment (markgreenburg) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: 👍 good catch

File: src/js/claims-status/containers/AppealsV2StatusPage.jsx [hunk 6]
```diff
@@ -12,15 +12,14 @@ import Docket from '../components/appeals-v2/Docket';
...
   const {
     events, alerts, status, docket, incompleteHistory, aod,
     active: appealIsActive,
     type: appealType
   } = appeal.attributes;
-  const currentStatus = getStatusContents(status.type, status.details);
-
-  // NB: 'details' doesn't do anything in getNextEvents for the time being
+  const currentStatus = getStatusContents(status.type, status.details, fullName);
+  // TODO: Might be a good idea to consolidate these two
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/js/claims-status/containers/AppealsV2StatusPage.jsx]: Yeah, just stood out to me that we're calling two separate functions in a row with the same arguments, which seems unnecessary. OTOH, it's more flexible, and there's nothing wrong with it.

[the remark this answers is not in the document]
Comment (cvalarida) [reply src/js/claims-status/containers/AppealsV2StatusPage.jsx]: Yeah, it seems like since they're unrelated, they probably shouldn't be joined together, y'know?

File: src/js/claims-status/utils/appeals-v2-helpers.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (cvalarida) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: Guess it don't make no nevermind anyhow. The browser eats up multiple spaces. :man_shrugging: 
Don't mind me.

**Review by cvalarida — APPROVED** [review]
Thanks for clearing up my questions. Looks like it's a go for me. My key is in and ready to turn. :rocket:

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 7]
```diff
@@ -229,29 +237,34 @@ export function getStatusContents(statusType, details) {
...
         because after completing the remand instructions from the Board, they couldn’t fully grant
         your appeal.</p>
       );
       break;
+    }
     case STATUS_TYPES.pendingHearingScheduling:
       contents.title = 'You’re waiting for your hearing to be scheduled';
       contents.description = (
-        <p>You requested a [TYPE] hearing on your Form 9. When your hearing is scheduled, you will
+        <p>You requested a {details.type} hearing on your Form 9. When your hearing is scheduled, you will
```
[the remark this answers is not in the document]
Comment (cmgiven) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: Yeah, we do need to transform the hearing type. Those will come back as `video`, `travel`, and `central_office`. Those should become: "videoconference", "travel board", and "Washington, DC central office".

`location` and `vsoName` are strings that can be used as is.
